frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -480,6 +480,11 @@ export const eventUsageLogic = kea<eventUsageLogicType>({
         reportInstanceSettingChange: (name: string, value: string | boolean | number) => ({ name, value }),
         reportAxisUnitsChanged: (properties: Record<string, any>) => ({ ...properties }),
         reportTeamSettingChange: (name: string, value: any) => ({ name, value }),
+        reportActivationSideBarShown: (
```

Comment:
I think it would also be good to record events for when people click the different actions within the sidebar so we can track how well the sidebar works for getting people to do the different things.

Reply:
Makes sense, done!